integration_test/third_party_apps_data/applications/varnish/metadata.yaml

```diff
@@ -0,0 +1,65 @@
+short_name: varnish
+long_name: Varnish HTTP Cache
+expected_metrics:
```

Comment:
Could we also add the expected logs to this file. See: https://github.com/GoogleCloudPlatform/ops-agent/pull/513

Reply:
added `expected_logs:` field